bindings/cs/MongoDB.Libmongocrypt/MongoDB.Libmongocrypt.targets

```diff
@@ -1,9 +1,10 @@
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ItemGroup Condition="Exists('packages.config') OR Exists('$(MSBuildProjectName).packages.config') OR Exists('packages.$(MSBuildProjectName).config')">
-    <Content Include="$(MSBuildThisFileDirectory)\..\runtimes\win\native\mongocrypt.dll">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-      <Visible>false</Visible>
-      <Link>mongocrypt.dll</Link>
-    </Content>
-  </ItemGroup>
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <ItemGroup Condition="$([MSBuild]::IsOsPlatform('Windows'))">
+        <Content Include="$(MSBuildThisFileDirectory)../runtimes/win/native/mongocrypt.dll">
+            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+            <Visible>false</Visible>
```

Comment:
let's mimic the same approach as we have in the driver. There is no real reason to make any different approach now. So I suggest to remove `Visisble` field,

Reply:
I'd say that I'm not very happy with making this change, because if we remove remove the `Visible` tag, net framework users will see a `mongocrypt.dll` hanging in their solution.
Done.